### PR TITLE
ADCM-3222 fix 500 While launching action with activatable config

### PR DESCRIPTION
**Pull request by DanBalalan**

### CODE REVIEW

File: python/cm/job.py [hunk 1]
```diff
@@ -191,7 +191,7 @@ def check_action_config(
     if obj.config:
         cl = ConfigLog.objects.get(obj_ref=obj.config, id=obj.config.current)
         obj_conf = cl.config
-        current_attr = cl.attr
+        current_attr = cl.attr or None  # convert {} (empty dict) to None
```
Comment (a-alferov) [inline python/cm/job.py:194]: What for ?

**Review by a-alferov — CHANGES_REQUESTED** [review]

File: python/cm/job.py (hunk 1 above)
Comment (DanBalalan) [reply python/cm/job.py:194]: to prevent KeyError in cm.adcm_config.check_attr when comparing attr's kaeys and current_attr's (cluster attr == {}) keys

**Comment by kuhella** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @kuhella! Build has been launched!

File: python/cm/job.py [hunk 2]
```diff
@@ -187,12 +187,11 @@ def check_action_config(
         return {}, {}
     if not conf:
         err("TASK_ERROR", "action config is required")
-    obj_conf, current_attr = None, None
+    obj_conf = None
     if obj.config:
-        cl = ConfigLog.objects.get(obj_ref=obj.config, id=obj.config.current)
-        obj_conf = cl.config
-        current_attr = cl.attr
```
Comment (a-alferov) [inline python/cm/job.py:194]: It is necessary to completely remove this block, and pass the configuration of the action to the function `variant.process_variant(obj, spec, conf)`

Comment (DanBalalan) [reply python/cm/job.py]: done

**Review by a-alferov — APPROVED** [review]

**Pull request merged**